### ISSUE TITLE
If this add-on is installed, is it Blender's default FBX exporter?

### ISSUE THREAD

**Issue by lajawi** [A-Ribeiro/CustomBlenderFBXExporter#11]
When I install this add-on, does Blender recognise it as an FBX exporter and uses it *instead* of the built-in one?

**Comment by FeuerTiger86** [comment]
No, but you can disable the built-in one. Or you could try replacing the files in the folder io_scene_fbx in the Blender installation itself (NOT the addons folder in AppData (on Windows), but in Program Files or where your Blender is installed).

**Comment by A-Ribeiro** [comment]
Hi.

I don't recommend to replace the default Blender's exporter...

And after install this addon, you will have two FBX exporters, and two Importers, the Blender defaults and this one.

here: https://github.com/A-Ribeiro/CustomBlenderFBXExporter/blob/main/images/import_export.jpg 

you can see the default exporter/importer and the custom exporter/importer.


P.S.: I'm a programmer and I would like to learn Blender... Could you suggest a good Blender tutorial?

**Comment by PhiMue86** [comment]
I suppose you know Blender Guru? I'd say that's a good starting point: https://www.youtube.com/playlist?list=PLjEaoINr3zgEPv5y--4MKpciLaoQYZB1Z

Once you got the basics, it depends on what you want to do. The workflows are quite different depending on the kind of assets you want to make, for games, animations, or render images, organic vs. hard surface, etc. The possibilities are endless, and so is the amount of things to learn. For example, Geometry Nodes I almost don't know at all, but I know that they can be used for amazing things.

**Comment by lajawi** [comment]
> No, but you can disable the built-in one. Or you could try replacing the files in the folder io_scene_fbx in the Blender installation itself (NOT the addons folder in AppData (on Windows), but in Program Files or where your Blender is installed).

> I don't recommend to replace the default Blender's exporter...
> 
> And after install this addon, you will have two FBX exporters, and two Importers, the Blender defaults and this one.

How unfortunate. You see, I've been thinkering around with Unity (game engine), and if you use `.blend` files as is as your 3D files in Unity, it uses Blender's exporter, hence the question.

But alas, the solution I hoped for, is no viable solution after all. Thank you for the help regardless!
